backend/models/time.py

```diff
@@ -165,10 +168,17 @@ def closest(self, origin, point_list):
         """
         distance_responses = self._measure_one_to_many_points(origin=origin, point_list=point_list)
 
-        min_idx, min_measurement = min(
-            enumerate(distance_responses),
-            key=operator.itemgetter(1)
-        )
+        try:
+            min_idx, min_measurement = min(
+                enumerate(distance_responses),
+                key=operator.itemgetter(1)
+            )
+        except TypeError:
+            logger.warning(
```

Comment:
I still don't understand why this block is ever entered. 

Maybe `_measure_one_to_many_points` should raise an error instead of returning meaningless distances?